examples/medplum-live-chat-demo/src/pages/HomePage.tsx

```diff
@@ -22,33 +23,112 @@ export function HomePage(): JSX.Element {
   // https://www.medplum.com/docs/tutorials/register
   const profile = useMedplumProfile() as Practitioner;
   const medplum = useMedplum();
+  const [thread, setThread] = useState<Communication>();
+
+  const profileRef = useMemo(() => createReference(profile as ProfileResource), [profile]);
```

Comment:
Super minor, but these `createReference` and `getReferenceString` calls probably do not need `useMemo` - they're all pretty cheap.